src/modules/app/components/inner-nav/inner-nav.jsx

```diff
@@ -5,6 +5,13 @@ import { mobileMenuStates } from 'modules/app/components/app/app'
 
 import Styles from 'modules/app/components/inner-nav/inner-nav.styles'
 
+import _, { isEqual } from 'lodash';
```

Comment:
Lodash is a fairly large lib, so only import the parts you need.

In this case, you’d want to do `import isEqual from ‘lodash/isEqual’` — this will keep us from having the _entire_ lodash lib included, but rather only `isEqual` and it’s dependencies.